src/plugins/intel_gna/src/transformations/gather_sinking_fuse.cpp

```diff
@@ -0,0 +1,141 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/gather_sinking_fuse.hpp"
+
+#include <openvino/cc/ngraph/itt.hpp>
+#include <transformations/utils/utils.hpp>
+#include <utility>
+
+#include "openvino/opsets/opset10.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "transformations/rt_info/gather_sinking_attr.hpp"
+#include "transformations/utils/gather_sinking_utils.hpp"
+
+using namespace ov;
+using namespace ov::opset10;
+using namespace ov::pass::pattern;
+using namespace ov::op::util;
+using namespace gather_sinking;
+using namespace ov::intel_gna::pass;
+using namespace ov::intel_gna::rt_info;
+
+namespace {
+bool GetGatherAxis(const std::shared_ptr<Gather>& gather, int64_t& axis) {
```

Comment:
Common issues: 
1) please fix code style (GetGatherAxis->get_gather_axis) 
2) all non-transformation specific helpers should be located in graph_utils